recipes/vapor/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "VAPOR" %}
```

Comment:
You need to open a PR on bioconda to remove their package at the same time that this one is added to conda-forge if it is the same package.